app/src/main/java/org/projectbuendia/client/ui/chart/PebbleExtension.java

```diff
@@ -192,6 +192,8 @@ public Object apply(Object input, Map<String, Object> args) {
                 for (Object item : (Collection) input) {
                     values.add(item instanceof ObsValue ? (ObsValue) item : null);
                 }
+            } else if (input instanceof ObsValue) {
```

Comment:
When is the input a single instance of ObsValue?  Can that call site use format_value instead of format_values?